packages/bot-web-ui/src/app/app-main.tsx

```diff
@@ -18,6 +19,14 @@ const App = ({ passthrough }: TAppProps) => {
         // of dynamic view height(vh) on mobile browsers for few scrollable components
         const vh = window.innerHeight;
         document.body.style.setProperty('--vh', `${vh}px`);
+
+        initSurvicate();
+        return () => {
+            const survicate_box = document.getElementById('survicate-box') || undefined;
```

Comment:
@maryia-matskevich-deriv I think we don't need to use undefined here as if the element is not found then it would return null which will be falsy anyway.

Reply:
done